src/Compute/Compute/ChangeLog.md

```diff
@@ -24,6 +24,7 @@
 * Updated the breaking change warning in `New-AzVM` and `New-AzVmss` regarding using the new versioned image aliases to indicate that certain aliases will be removed next breaking change release.
 * Updated the `Get-AzVMRunCommand` to include the `ProvisioningState` value. Fix [#21473]
 * Updated Azure.Core to 1.31.0.
+* Added new switch parameter `OSImageScheduledEventEnabled` and string parameter `OSImageScheduledEventNotBeforeTimeoutInMinutes` to the cmdlets `New-AzVmssConfig` and `Update-AzVmss`.
 
```

Comment:
I think this will conflict with main after the release branch is merged back. So please move this to the top.